deps/chakrashim/src/v8objecttemplate.cc

```diff
@@ -780,14 +780,8 @@ JsValueRef CHAKRA_CALLBACK Utils::DefinePropertyCallback(
   }
 
   if (result == JS_INVALID_REFERENCE) {
-    // No interception took place; fall back to default behavior
-    JsPropertyIdRef propertyIdRef;
-    if (jsrt::GetPropertyIdFromName(prop, &propertyIdRef) != JsNoError) {
-      return jsrt::GetFalse();
-    }
-
     bool result = false;
-    if (JsDefineProperty(object, propertyIdRef, descriptor, &result)
+    if (JsObjectDefineProperty(object, prop, descriptor, &result)
```

Comment:
I believe that this isn't quite right; Right now this code path isn't going to be used until we fix our handling of `cm.runInContext`, but when it is used it is perfectly allowable for `prop` to be a `Symbol`, which would not be handled with this change.

Reply:
Right, Symbol support should be in master no time.